nginx.conf

```diff
@@ -21,9 +21,13 @@ server {
 
   rewrite ^/clientdev/(.*)$ /development/sdk-dev/$1$is_args$args redirect;
   rewrite ^/integrations/(.*)$ /workflow/integrations/$1$is_args$args redirect;
-  rewrite ^/internal/(api|docs|contributing|environment)/?$ /development/server/$1/$is_args$args redirect;
-  rewrite ^/development/contribute/(.*)$ /development/server/$1$is_args$args redirect;
-  
+
+  # develop redirects
+  rewrite ^/internal/(api|docs|contributing|environment)/?$ https://develop.sentry.dev/$1/$is_args$args redirect;
+  rewrite ^/development/contribute/(.*)$ https://develop.sentry.dev/ redirect;
+  rewrite ^/development/server/(.*)$ https://develop.sentry.dev/ redirect;
```

Comment:
Isn't this missing `/$1/`?

Reply:
Bunch of those urls wouldn't map up (or no longer will exist, tho I should probably delete them in this PR..) so I figured safer to send to root

Reply:
`/internal/environment/` currently redirects to `https://docs.sentry.io/development/server/environment/` and will now redirect to `https://develop.sentry.dev/environment/` but `/development/server/environment` where this page actually was goes just to the root.

Reply:
Might as well just make the old `/internal/` redirects go to root then as well.

Reply:
good point